nginx/nginx.conf

```diff
@@ -0,0 +1,16 @@
+events {
+  worker_connections 1024; # increase if you have lots of clients
+  accept_mutex off; # "on" if nginx worker_processes > 1
+}
+
+http {
```

Comment:
I think you need (something similar to) the following here to resolve the css issue:

```nginx
include /etc/nginx/mime.types;
```